### ISSUE TITLE
 category axes bullets tooltips overlaps

### ISSUE THREAD

**Issue by paxapy** [amcharts/amcharts4#1464]
i've noticed that when I use two category axes on line series and circle bullets, tooltips kinda got overlapped:
https://codepen.io/paxapy/pen/MMMvOL
it's really hard to see tooltips that actually related to hovered bullet.
is there something wrong with my config, or workaround somehow?

**Comment by martynasma** [comment]
Try disabling cursor-activated tooltip:

```
series.cursorTooltipEnabled = false;
```

This way a relevant tooltip will only be shown when you hover specific bubble.

**Comment by paxapy** [comment]
@martynasma thanks, seems like it working perfectly

**Comment by tapiamcclung** [comment]
After finding #1464, I redirected here. 
Bringing this back up, I added a cursor to highlight stuff while displaying the tooltip on hover.
Forked the basic heatmap and added the appropriate cursor stuff and added this from above
```
series.cursorTooltipEnabled = false;
```
It works, except the first tooltip does not show up... Its <g> element's visibility remains "hidden" and its opacity is 0. Hovering on subsequent cells shows normal functionality.
Here's a pen:
https://codepen.io/tapiamcclung/pen/GRpvvpm
